front/components/app/Deploy.tsx

```diff
@@ -76,8 +76,8 @@ export default function Deploy({
   };
 
   // Copy the cURL request to the clipboard
-  const handleCopyClick = () => {
-    navigator.clipboard.writeText(cURLRequest(true));
+  const handleCopyClick = async () => {
+    await navigator.clipboard.writeText(cURLRequest(true));
```

Comment:
IMO, it's better to have `.catch` here, because no one is going to `await` `handleCopyClick`, so if it fails it will still be an unhandled promise rejection

Reply:
Hmm, I am not sure I fully understand what you mean.
This function is an async so I think we should await it. Otherwise we might show "copied!" even though it's false.